source/binary.cpp

```diff
@@ -732,6 +739,41 @@ spv_result_t Parser::setNumericTypeInfoForType(
   return SPV_SUCCESS;
 }
 
+spv_result_t Parser::verifyLiteralUpperBits(
+    const spv_parsed_operand_t& parsed_operand, uint32_t word) {
+  // Verify the upper bits of literal numbers not multiple of 32 bits
```

Comment:
I'm not sure I understand this comment.  This is verifying that the upper bits are either not set or sign-extended, right?  Not sure how that is considered "multiple of 32 bits".